core-kotlin-modules/core-kotlin-strings-3/README.md

```diff
@@ -0,0 +1,7 @@
+## Core Kotlin Strings
+
+This module contains articles about core Kotlin strings.
+
+### Relevant articles:
+
+- [Removing Whitespace from Strings](https://www.baeldung.com)
```

Comment:
I think we can leave this off for now and it'll be added back later when the article is processed.

```suggestion
```